components/fxa-client/examples/migration.rs

```diff
@@ -15,8 +14,22 @@ fn main() {
     let k_sync: String = prompt_string("k_sync").unwrap();
     println!("Enter kXCS (hex-string):");
     let k_xcs: String = prompt_string("k_xcs").unwrap();
-    fxa.migrate_from_session_token(&session_token, &k_sync, &k_xcs, true)
-        .unwrap();
-    println!("WOW! You've been migrated.");
+    let migration_result =
+        match fxa.migrate_from_session_token(&session_token, &k_sync, &k_xcs, true) {
+            Ok(migration_result) => migration_result,
+            Err(err) => {
+                println!("Error: {}", err);
+                // test offline behaviour
```

Comment:
Unless this gets run in CI on a regular basis, I'm not sure we can claim it as a "test" ;-)